examples/boto3/list_unordered.py

```diff
@@ -22,4 +22,4 @@
 
 # getting an unordered list of objects is an extension to AWS S3 API
 
-print(client.list_objects(Bucket=bucketname, AllowUnordered=True))
+print(client.list_objects(Bucket=bucketname))
```

Comment:
we want to demonstrate our extension that allow for unordered listing

Reply:
I was testing it on AWS Cli version 1 which does not have the AllowUnordered=True option. I will make the necessary changes.